operations/customers.md

```diff
@@ -225,10 +225,10 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `MergeTargetId` | string | optional | Unique identifier of the account ([Customer](customers.md#customer)) to which this customer is linked. |
 | `IsActive` | boolean | required | Whether the customer record is still active. |
 | `PreferredSpaceFeatures` | array of [Resource Feature Classification](_objects.md#resource-feature-classification) | required | A list of room preferences, such as view type, bed type, and amenities. |
-| ~~`Passport`~~ | ~~[Identity document](customers.md#identity-document)~~ | ~~optional~~ | **Deprecated!** Use record from `Documents` with relevant `CustomerId` instead.|
-| ~~`IdentityCard`~~ | ~~[Identity document](customers.md#identity-document)~~ | ~~optional~~ | **Deprecated!** Use record from `Documents` with relevant `CustomerId` instead.|
-| ~~`Visa`~~ | ~~[Identity document](customers.md#identity-document)~~ | ~~optional~~ | **Deprecated!** Use record from `Documents` with relevant `CustomerId` instead.|
-| ~~`DriversLicense`~~ | ~~[Identity document](customers.md#identity-document)~~ | ~~optional~~ | **Deprecated!** Use record from `Documents` with relevant `CustomerId` instead.|
+| ~~`Passport`~~ | ~~[Identity document](customers.md#identity-document)~~ | ~~optional~~ | **Deprecated!** Use `identityDocuments/getAll` to fetch identity documents.|
```

Comment:
Please use [Get all identity documents](identitydocuments.md#get-all-identity-documents) rather than `identityDocuments/getAll`, as per standard practice.